resource_booking/migrations/16.0.1.0.0/post-migration.py

```diff
@@ -0,0 +1,14 @@
+from openupgradelib import openupgrade
+
+
+def _update_attendance_hour_to(env):
+    # 23:59 -> 24:00
+    lines = env["resource.calendar.attendance"].search(
```

Comment:
Why is this needed?

Reply:
I have added a comment in the method to try to explain why this is necessary.